agent/src/util/config_action.rs

```diff
@@ -326,8 +323,8 @@ mod config_action_tests {
         let _ = env_logger::builder().is_test(true).try_init();
         let path_to_config = "../test/yaml/config-a.yaml";
         let config_yaml = fs::read_to_string(path_to_config).expect("Unable to read file");
-        let config: KubeAkriConfig = serde_yaml::from_str(&config_yaml).unwrap();
-        let config_name = config.metadata.name.clone();
+        let config: Configuration = serde_yaml::from_str(&config_yaml).unwrap();
+        let config_name = config.metadata.name.clone().unwrap();
```

Comment:
why do we need to unwrap the name?

Reply:
nevermind ... i see it in k8s-openapi: `pub name: Option<String>` ... strange to imagine a resource without a name.

Reply:
Right? `k8s_openapi` updated to put `Option` around almost everything. I dont see a case when a `Configuration` `Pod` etc wouldn't have a name.